src/main/java/laddergame/domain/user/Users.java

```diff
@@ -0,0 +1,49 @@
+package laddergame.domain.user;
+
+import com.google.common.collect.Lists;
+import laddergame.domain.ladder.exception.UserNotFoundException;
+
+import java.util.List;
+
+public class Users {
+
+    private final List<User> users;
+
+    private Users(final String namesWithComma) {
+        this.users = initUsers(namesWithComma);
+    }
+
+    public static Users of(final String namesWithComma) {
+        return new Users(namesWithComma);
+    }
+
+    private List<User> initUsers(final String namesWithComma) {
+        List<User> users = Lists.newArrayList();
+        String[] names = namesWithComma.split(",");
+        for (int position = 0; position < names.length; position++) {
+            users.add(User.of(names[position], position));
+        }
+        return users;
+    }
+
+    public int getSize() {
+        return users.size();
+    }
+
+    public User getUserOf(final Position position) {
+        return users.stream().filter(user -> user.isPositionOf(position))
+            .findAny()
+            .orElseThrow(UserNotFoundException::new);
+    }
+
+    public Position getPositionOf(final String userName) {
+        return users.stream().filter(user -> user.isEqualsName(userName))
+            .findAny()
+            .map(User::getPosition)
+            .orElseThrow(UserNotFoundException::new);
+    }
+
+    public List<User> getUsers() {
+        return users;
```

Comment:
```suggestion
        return Collections.unmodifiableList(users);
```

불변 리스트로 바꿔서 리턴하는게 좋아보여요